File: javascript/temperatura.js
const resposta = prompt("Voce vai digitar a temperatura em qual escala (C/F)?").toUpperCase();
let c, f;

if (resposta == 'F') {
    f = parseFloat(prompt("Digite a temperatura em Fahrenheit"));

    c = (f - 32) * 5 / 9;

    console.log("Temperatura equivalente em Celsius: %.2f", c);
} else {
    c = parseFloat(prompt("Digite a temperatura em Celsius"));

    f = (9 * c / 5) + 32;

    console.log("Temperatura equivalente em Fahrenheit: %.2f", f);
}

File: python/temperatura.py
escala = input("Voce vai digitar a temperatura em qual escala (C/F)? ").upper()

if escala == "F":
    fahrenheit = float(input("Digite a temperatura em Fahrenheit: "))

    celsius = 5 / 9 * (fahrenheit - 32)
    print(f"Temperatura equivalente em Celsius: {celsius:.2f}")
else:
    celsius = float(input("Digite a temperatura em Celsius: "))

    fahrenheit = 9 / 5 * celsius + 32
    print(f"Temperatura equivalente em Fahrenheit: {fahrenheit:.2f}")
